examples/instability_vs_pyheadtail/000_instability_vs_pyheadtail.py

```diff
@@ -0,0 +1,286 @@
+# copyright ############################### #
+# This file is part of the Xtrack Package.  #
+# Copyright (c) CERN, 2021.                 #
+# ######################################### #
```

Comment:
Is this needed?

Reply:
This is needed on all files, but this statement (and a few similar other ones, in a handful of files) were not up-to-date, so removed, and to be replaced thanks to an automatic script.
Issue #26 created.